### ISSUE TITLE
Possible issue with Instagram login

### ISSUE THREAD

**Issue by jth0** [ttscoff/Slogger#417]
While installing & configuring Slogger for the first time, I'm encountering the following error when I try to enable Instagram:

{"code": 403, "error_type": "OAuthForbiddenException", "error_message": "You are not a sandbox user of this client"}

This occurs after Slogger opens localhost:4567 and I click the link to connect, then log into Instagram (or when clicking the link after I've logged into the website.)

I am not sure what to try, but I didn't see any instructions for any specific action I need to take.

**Comment by ttscoff** [comment]
This plugin is by @rjames86, I believe, so he'll have to weigh in on any API changes. Personally, I use IFTTT to send my Instagram posts to Dropbox, where slogger.rb picks them up.

**Comment by rjames86** [comment]
Unfortunately I lost access to the Instagram API due to their new policy. Their terms disallow automations to pull their images. Sorry :(

I can do a pull request to remove the plugin if needed.

**Comment by ttscoff** [comment]
Yeah, we probably should, mostly for the sake of avoiding support 
tickets on it…

-Brett
